audio/espeakedit/files/patch-espeakedit-mac.diff

```diff
@@ -0,0 +1,96 @@
+diff -ruEN orig/Makefile src/Makefile
+--- orig/Makefile	2020-06-25 21:54:53.000000000 +0700
++++ src/Makefile	2020-06-26 09:34:19.000000000 +0700
+@@ -10,17 +10,18 @@
+ 
+ OBJS=$(patsubst %.cpp,%.o,$(SRCS))
+ 
+-WX_LIBS = -pthread   `wx-config --libs`
++WX_LIBS=`/opt/local/Library/Frameworks/wxWidgets.framework/Versions/wxGTK/2.8/bin/wx-config --libs`
++
++WX_CFLAGS=`/opt/local/Library/Frameworks/wxWidgets.framework/Versions/wxGTK/2.8/bin/wx-config --cflags`
+ 
+ LIBS=-lstdc++ -lportaudio
+-#LIBS=-lstdc++ /usr/lib/x86_64-linux-gnu/libportaudio.so.2
+ 
+-CPPFLAGS =  -Wall -g -fexceptions `wx-config --cflags`
++CPPFLAGS= -Wall -g -fexceptions $(WX_CFLAGS)
+ 	-I/usr/include/wx-2.8 \
+ 	-DGTK_NO_CHECK_CASTS -D__WXGTK__ -D_FILE_OFFSET_BITS=64 -D_LARGE_FILES
+ 	-D_LARGEFILE_SOURCE=1 -DNO_GCC_PRAGMA -D_ESPEAKEDIT
+ 
+-CXXFLAGS = -O2  -Wall -fexceptions `wx-config --cflags` \
++CXXFLAGS= -O2 -Wall -fexceptions $(WX_CFLAGS) \
+    -DGTK_NO_CHECK_CASTS -D__WXGTK__ -D_FILE_OFFSET_BITS=64 -D_LARGE_FILES -D_LARGEFILE_SOURCE=1 -DNO_GCC_PRAGMA -D_ESPEAKEDIT
+ 
+ all: espeakedit
+@@ -29,7 +30,7 @@
+ 	$(CXX) $(CXXFLAGS) -I. -c -fno-exceptions $<
+ 
+ espeakedit: $(OBJS)
+-	$(CXX) -o $@ $(OBJS) $(LIBS) $(WX_LIBS)
++	$(CXX) -o $@ $(OBJS) $(LIBS) -pthread $(WX_LIBS)
+ 
+ clean:
+ 	rm -f *.o *~
+diff -ruEN orig/espeakedit_test src/espeakedit_test
+--- orig/espeakedit_test	1970-01-01 07:00:00.000000000 +0700
++++ src/espeakedit_test	2020-06-27 22:28:50.000000000 +0700
+@@ -0,0 +1,14 @@
++#!/bin/bash
++
++DISPLAY=:55
++/opt/X11/bin/xvfb :55 -screen 0 1024x768x24 &
```

Comment:
Don't require the user to have installed things in /opt/X11. We have X11 in MacPorts; hopefully we have a port that provides xvfb and you can depend on and use that instead.

Reply:
I can't find xvfb in any of the MacPorts X11 or xorg packages.

Reply:
Ok. But you have /opt/X11/bin/xvfb on your system? Where did it come from?

Reply:
I assume it came from the Xquartz main installer pkg. I found xvfb listed on https://www.xquartz.org/Developer-Info.html but it it looks like it would take a new xorg-server-extras port to build it for MacPorts.

Reply:
Shall I just remove the test block?